src/fivetran/opsGenie/lambdaHandler.ts

```diff
@@ -0,0 +1,70 @@
+import { fiveTranRequest, context } from '../../commands/fiveTranInterface'
+import { listOpsGenieAlerts } from '../../commands/opsGenieCommands'
+import { secureEnvironmentVariable } from '../../aws/secretsClient'
+
+let opsGenieApiKey: string
+
+
+export async function handler(request: fiveTranRequest, context: context) {
+  console.log('request', request)
+  console.log('context', JSON.stringify(context, null, 2))
+  return await update(request.state.lastUpdate)
+}
+async function update(state: string) {
+  const decryptRequest: Promise<void> = decryptSlackSecret()
+  await decryptRequest
+  opsGenieApiKey = process.env.OPS_GENIE_SECRET || ''
+
+  // Fetch records using api calls
+
+  const insertTransactions = await apiResponse(state)
+
+  console.log(insertTransactions.length)
+  // if opsGenie returns empty array
+  if(insertTransactions.length == 0){
```

Comment:
```suggestion
  const hasData = insertTransactions.length > 0
  if(hasData){
```

Recommend reversing the logic to aid readability:
- Avoid double negative logic (confusing)
- Prefer affirmative statements (somthing is true) - easier to grok - reduces cognitive load.

Reply:
logic improved